bin/nrnivmodl_makefile_cmake.in

```diff
@@ -23,10 +23,16 @@ incdir := ${ROOT}/@CMAKE_INSTALL_INCLUDEDIR@
 datadir:= ${ROOT}/@CMAKE_INSTALL_DATADIR@
 datadir_lib := ${ROOT}/@CMAKE_INSTALL_DATADIR@/lib
 
+# For a static build with Interviews need -linterviews -lreadline
+IV_LINK =
+ifeq (@IV_ENABLE_SHARED@@NRN_ENABLE_INTERVIEWS@,OFFON)
+    IV_LINK = -linterviews -lreadline
```

Comment:
is there a cmake variable `Readline_LIBRARY` that should be used?

Reply:
This started out as a very specific issue regarding a static build on the mac for NIA (Neurons In Action) with no MPI or Python. I quickly observed that nrnivmodl linking required the addition of "-linterviews -lreadline" to get a static special.
Given the CI issues, this is a classic case of a change inadvertently breaking things in unrelated contexts, particularly wheels. I'm trying to put together a more focused fix.

Reply:
I'm at a loss as to why this small change results in CI failure on windows (not on my windows10 virtualbox guest).